druntime/src/core/sys/posix/sys/stat.d

```diff
@@ -1656,7 +1651,6 @@ else version (CRuntime_Bionic)
 }
 else version (CRuntime_Musl)
 {
-    alias __mode_t = uint;
```

Comment:
Where does musl typedef `__mode_t`? That it is currently in target specific block suggests that it perhaps shouldn't be set for all blindly. Does only removing this one line also work?

Reply:
musl does not define `__mode_t`. I don't know why this definition is here at all.

Reply:
Note that musl does not even define `mode_t`, that comes from `linux-api-headers` (which comes from the kernel tree).

Reply:
Yeah, just kill the Musl alias only then, no need to move the existing definitions.
```
$ git grep __mode_t
src/core/sys/posix/sys/stat.d:            alias __mode_t = uint;
src/core/sys/posix/sys/stat.d:            __mode_t st_mode;
src/core/sys/posix/sys/stat.d:            alias __mode_t = uint;
src/core/sys/posix/sys/stat.d:            __mode_t st_mode;
src/core/sys/posix/sys/stat.d:            alias __mode_t = uint;
src/core/sys/posix/sys/stat.d:            __mode_t st_mode;
src/core/sys/posix/sys/stat.d:            alias __mode_t = uint;
src/core/sys/posix/sys/stat.d:            __mode_t st_mode;
src/core/sys/posix/sys/stat.d:            alias __mode_t = uint;
src/core/sys/posix/sys/stat.d:            __mode_t st_mode;
src/core/sys/posix/sys/stat.d:            alias __mode_t = uint;
src/core/sys/posix/sys/stat.d:            __mode_t st_mode;
src/core/sys/posix/sys/stat.d:            alias __mode_t = uint;
src/core/sys/posix/sys/stat.d:            __mode_t st_mode;
src/core/sys/posix/sys/stat.d:    alias __mode_t = uint;
```

Reply:
Thanks, done.